ParalayoutTests/UIViewSizingTests.swift

```diff
@@ -1,9 +1,17 @@
 //
```

Comment:
This could probably use some more tests.

- [ ] Add tests for `sizeToFit*` methods